package.json

```diff
@@ -96,6 +96,9 @@
     "build:version": "node ./scripts/build-version.js",
     "build:prebuild": "node ./scripts/prebuild.js",
     "build": "yarn build:prebuild && yarn build:app && yarn build:version",
+    "install:deps": "yarn install:deps:excalidraw && yarn install:deps:utils",
+    "install:deps:excalidraw": "yarn --cwd src/packages/excalidraw install:deps",
+    "install:deps:utils": "yarn --cwd src/packages/utils install:deps",
```

Comment:
@DanielJGeiger even these could be removed for the reasons mentioned earlier - we will be moving to mono repo soon so each package will be treated individually in that case.

Reply:
Removed in eb76191.